index.js

```diff
@@ -13,7 +13,7 @@ function traceCaller (pinoInstance) {
 
   function asJson (...args) {
     args[0] = args[0] || Object.create(null)
-    args[0].caller = Error().stack.split('\n')[STACKTRACE_OFFSET].substr(LINE_OFFSET)
+    args[0].caller = Error().stack.split('\n').filter(s => !s.includes('node_modules/pino'))[STACKTRACE_OFFSET].substr(LINE_OFFSET)
```

Comment:
pino might be a dependency of a dependency in some case. Also, I'm not sure if this would match paths on windows.

Reply:
> I'm not sure if this would match paths on windows.

i added a check for windows fs paths.

> pino might be a dependency of a dependency in some case

not really sure of what you mean, can you be more specific?